app/_includes/footer.html

```diff
@@ -61,19 +125,9 @@ <h4>Legal</h4>
   <section class="legal">
     <ul>
       <li>
-        <span class="mashape-footer-content">&copy; Kong Inc. {{ site.time | date: '%Y' }}</span>
-      </li>
-      <li>
-        <div class="social-link">
-          <a href="https://github.com/Kong" title="GitHub"><i aria-label="GitHub" class="fa fa-github" aria-hidden="true"></i></a></div>
-        <div class="social-link">
-          <a href="https://www.facebook.com/konghq/" title="Facebook"><i aria-label="Facebook" class="fa fa-facebook-official" aria-hidden="true"></i></a></div>
-        <div class="social-link">
-          <a href="https://www.meetup.com/topics/kong/all/" title="Meetup"><img aria-label="Meetup" alt="Meetup" src="/assets/images/footer/icn-meetup-black.svg"></a>
-        </div>
-        <div class="social-link">
-          <a href="https://twitter.com/thekonginc" title="Twitter"><i aria-label="Twitter" class="fa fa-twitter" aria-hidden="true"></i></a>
-        </div>
+        <span class="mashape-footer-content"
+          >&copy; Kong Inc. {{ site.time | date: '%Y' }} &nbsp; <a href="http://konghq.com/terms/">Terms</a><b>•</b><a href="http://konghq.com/privacy/">Privacy</a></span
+        >
```

Comment:
The closing `>` on 129 and 130 seem out of place; wish I could suggest multiple lines to save you time.